src/page.js

```diff
@@ -6,6 +6,28 @@ export default class Page {
         this.target = 'page$' + pageId;
         this.phantom = phantom;
     }
+
+    on(event, runOnPhantom, callback) {
```

Comment:
It isn't the same as property. Property would override the previous listener (if that isn't just a setter). 

I have an event pool in the shim file, and every time you ad an event, the listener is added to the pull.

PS: the doc will be added in a while